sklearn/linear_model/coordinate_descent.py

```diff
@@ -786,19 +786,19 @@ class Lasso(ElasticNet):
 
     Parameters
     ----------
-    alpha : float, optional
-        Constant that multiplies the L1 term. Defaults to 1.0.
+    alpha : float, default 1.0.
```

Comment:
I think we should do
``parameter_name : type (default=x)``
comma before parenthesis is redundant.